libmscore/bracket.cpp

```diff
@@ -471,6 +471,10 @@ QVariant Bracket::propertyDefault(Pid id) const
 
 void Bracket::undoChangeProperty(Pid id, const QVariant& v, PropertyFlags ps)
 {
+    if (id == Pid::COLOR) {
```

Comment:
please tell me why you need it?

Reply:
Elements in the palettes panel must change their colors accordingly to our qml theme. I added some code to implement this